packages/react-events/src/FocusScope.js

```diff
@@ -157,4 +329,50 @@ const FocusScopeResponder = {
   },
 };
 
-export default ReactDOM.unstable_createEvent(FocusScopeResponder, 'FocusScope');
+export const FocusScopeEventComponent = ReactDOM.unstable_createEvent(
+  FocusScopeResponder,
+  'FocusScope',
+);
+
+const FocusScopeContext: React.Context<FocusManager> = React.createContext();
+
+export function FocusScope(props: FocusScopeProps) {
+  let internalFocusManager: ?FocusManager;
```

Comment:
Is it intentional that this gets reset every time `FocusScope` renders? Maybe this should be stored in a ref?

Reply:
That's a good suggestion